css/responsive/desktop.styl

```diff
@@ -6,6 +6,7 @@ $gutter = 32px
   
 .col-l
   padding: 0 ($gutter / 2)
+  margin-bottom: ($gutter / 2)
```

Comment:
in that case I think it is better to declare a `.col.col-no-gutter` class to override as the majority of usage should be proper with full gutter and we can save a lot of extra markup and made the code a lot more readable